packages/wallet-service/src/nodeConfig.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { getUnixTimestamp } from '@src/utils';
+import { ServerlessMysql } from 'serverless-mysql';
+import { getVersionData, updateVersionData } from '@src/db';
+import { FullNodeVersionData, FullNodeApiVersionResponse } from '@src/types';
+import fullnode from '@src/fullnode';
+
+const VERSION_CHECK_MAX_DIFF = 60 * 60; // 1 hour
```

Comment:
Use an env variable for this? Was it hardcoded before?

Reply:
It was hardcoded but I can move to a configuration